modules/op-behavior-of-approvaltask-with-groups-and-users.adoc

```diff
@@ -0,0 +1,71 @@
+// This module is included in the following assemblies:
+// * create/using-manual-approval.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-behavior-of-approvaltask-with-groups-and-users_{context}"]
+= Behavior of ApprovalTask with groups and users
+
+The `ApprovalTask` custom resource (CR) provides a manual approval gate for Tekton pipelines. You can configure an approval task to require approvals from individual users, groups, or a combination of both. When a `PipelineRun` reaches an approval step, the Manual Approval Gate controller creates the corresponding `ApprovalTask` resource and pauses the pipeline until the required number of approvals is received.
+
+You can assign approval responsibility to a group. When a group is configured as an approver, the controller tracks individual approvals from group members and updates the group’s overall approval state automatically.
+
+.Example: Creating an `ApprovalTask` with a group approver
```

Comment:
need to add that its created by controller here too. Maybe
"Initial ApprovalTask created by the controller with a group approver (no approvals recorded yet)" or 
"ApprovalTask with a group approver created by the controller before any approvals are recorded"
something along these lines to make it explicit